src/image-marker.js

```diff
@@ -0,0 +1,56 @@
+import { $ } from 'clappr'
+import StandardMarker from './standard-marker'
+
+/*
+ * An implementation of an image Marker, which can show an image.
+ */
+export default class ImageMarker extends StandardMarker {
+
+  /*
+   * time: the time in seconds that this marker represents
+   * tooltipImage: the image to be shown (optional)
+   * width: width for the image (Default: 200px)
+   * height: height for the image (Default: auto)
+   */
+  constructor (time, tooltipImage, width, height) {
+    super(time, tooltipImage)
+    this._tooltipText = tooltipImage || null
+    this._width = width || '200px'
+    this._height = height || 'auto'
+    this._$marker = this._buildMarkerEl()
+    this._$tooltip = this._buildTooltipEl()
+    this._addListeners()
+  }
+
+  _buildTooltipEl () {
+    if (!this._tooltipText) {
+      return null
+    }
+    return $('<div />').addClass('standard-tooltip').append("<img src='" + this._tooltipText + "' width='" + this._width + "' height='" + this._height + "' />")
```

Comment:
I think the following would be better setting the width and height with CSS instead of attributes. Also setting the source this way will do any escaping if needed.

```javascript
var $img = $('<img />').attr('src', this._tooltipText).css({
  width: this._width,
  height: this._height
})
$img.one('load',  this.notifyTooltipChanged)
return $('<div />').addClass('standard-tooltip').append($img)
```

The extra line should make sure that the tooltip resizes and is positioned correctly after the image loads if it loads after the user has triggered the marker.